site/src/components/WorkspaceActions/WorkspaceActions.tsx

```diff
@@ -171,6 +171,6 @@ const useStyles = makeStyles((theme) => ({
     },
   },
   popoverPaper: {
-    padding: `${theme.spacing(2)}px ${theme.spacing(3)}px ${theme.spacing(3)}px`,
+    padding: `${theme.spacing(1)}px ${theme.spacing(2)}px ${theme.spacing(1)}px`,
```

Comment:
❓ Since this affects `top | horizontal | bottom` and the person before intentionally added extra padding to the bottom (3 instead of 2), I wonder if this will have negative effects when there are more items in the dropdown? 🤔

cc @BrunoQuaresma @Kira-Pilot

Reply:
You can check in the storybook: WorkspaceActions -> Canceled

Reply:
@jsjoeio I don't see any issues looking at the storybook!